low-level/framework/messaging/ingress_processor.py

```diff
@@ -193,9 +195,21 @@ def _process_msg(self, body):
             self._check_debug(message)
             self._log_debug("_process_msg, ingressMsg: %s" % ingressMsg)
 
-            self._send_to_msg_handler(msgType, message, uuid)
+            # Compare hostname from the request to determine
+            # if request is meant for the current node
+            node_identifier = ingressMsg.get("node_identifier")
```

Comment:
So CSM will add node_identifier key with hostname as value ?

Reply:
We will need to communicate this to the end users (HA, CSM). I am thinking of some way to keep in backward compatible so if someone does not include the `node_identifier`, the request should proceed in old way.